common/app/model/Section.scala

```diff
@@ -15,9 +16,16 @@ object Section {
     val keywordIds: Seq[String] = frontKeywordIds(id)
     val keywordSponsorship = KeywordSponsorshipHandling(id, adUnitSuffix, keywordIds)
 
+    // The PI campaign will run for one year, during which all the related pages must provide a sticky
+    // banner at the top that sticks all the way through, therefore overriding other config flags
+    // such as isAdvertisementFeature
+    lazy val isPersonalInvestmentsCampaign: Boolean = keywordIds.exists(t => t.endsWith("/personal-investments"))
+    lazy val isPersonalInvestmentsCampaignRunning: Boolean = DateTime.now().isBefore(new DateTime(2017, 4, 26, 0, 0))
+
     val javascriptConfigOverrides: Map[String, JsValue] = Map(
         "keywords" -> JsString(webTitle),
         "keywordIds" -> JsString(keywordIds.mkString(",")),
+        "hasSuperStickyBanner" -> JsBoolean(isPersonalInvestmentsCampaign && isPersonalInvestmentsCampaignRunning),
```

Comment:
Perhaps this condition could be encapsulated in a `PersonalInvestmentsCampaign` object that `Section` and `PressedPage` both use, to avoid repetition.